asreview/webapp/api/projects.py

```diff
@@ -1125,23 +1132,54 @@ def _get_stats(project, include_priors=False):
 
         n_records = len(as_data)
 
+        # Get labels without priors
+        labels_without_priors = s.get_labels(priors=False)
+
```

Comment:
This causes the tests to fail I think. This should be inside the context manager.